pkg/adapter/incoming_test.go

```diff
@@ -21,10 +23,27 @@
 	"go.uber.org/zap"
 	zapobserver "go.uber.org/zap/zaptest/observer"
 	"gotest.tools/v3/assert"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	rtesting "knative.dev/pkg/reconciler/testing"
 )
 
+const fakePrivateKey = `-----BEGIN RSA PRIVATE KEY-----
```

Comment:
not our security team robot is going to bug us when they sees this 🙄

Reply:
its just fake data 
does it create issue :thinking:

Reply:
It is not a good advice to keep sensitive data in source, even if it is a fake data.
We have to give explanation each time when it hits security audit.
better we can generate it dynamically